### DISCUSSION TITLE
On running x.py

### DISCUSSION THREAD

**Discussion by mTvare6** [mTvare6/hello-world.rs#327] [Announcements]
Running the x.py script on my machine(a mac), for 12.09 seconds creates 63339 lines, so for each second, apprx 5238 lines in the beginning, the later would go even slower due to higher computation required for finding words nearing billions. At this rate for the script to complete(9,223,372,036,854,775,807 lines), it would require  apprx, 20,218,180,145 days, or 55,392,274 years. And the same file takes 5Mb, thus for 9,223,372,036,854,775,807 lines it would take 711,075,965,480Gb(if all names are as small as in 1->62k range but expotentially more considering longer words will be required).

Thus my kind request is that someone spend time trying to complete the file, at least by parts modifying to get specific ranges [x.py](https://github.com/mTvare6/hello-world.rs/blob/master/x.py) so that we might be able to see full power of the rust compiler in compiling such files, to show the superiority of rust. 

Last but not least, thanks to #78 to inspire me to build such great artifacts and my comment on [24 Sep 2021](https://github.com/mTvare6/hello-world.rs/pull/78#issuecomment-926594829)

**Comment by alexisLefebvre** [comment]
> Running the x.py script

Python is slow :snail:, please run only blazing fast Rust code :rocket:.

**Reply by mTvare6** [reply]
I use rustpython, https://github.com/RustPython/RustPython

**Reply by vikramdurai** [reply]
> I use rustpython, https://github.com/RustPython/RustPython

Stop write python !,. very bad...average American . 去你的 ancestor cry ! 
\- [Zhang Yong](https://reddit.com/r/ZhangYong)
